contracts/examples/DeviceNFT.sol

```diff
@@ -8,15 +8,15 @@ contract DeviceNFT is ERC721Upgradeable, OwnableUpgradeable {
     event MinterConfigured(address indexed minter, uint256 minterAllowedAmount);
     event MinterRemoved(address indexed minter);
     event MinterAllowanceIncremented(address indexed owner, address indexed minter, uint256 allowanceIncrement);
-    event SetBastURI(string uri);
+    event SetBaseURI(string uri);
 
     mapping(address => bool) internal minters;
     mapping(address => uint256) internal minterAllowed;
     string internal uri;
     uint256 public total;
 
     function initialize(string memory _name, string memory _symbol) external initializer {
-        __Ownable_init();
+        __Ownable_init(msg.sender);
```

Comment:
Because the ioID contracts using openzeppelin v4.9.5, so can't change to ` __Ownable_init(msg.sender);`, it's must be `__Ownable_init();` at this repo, but developer can develop their owned DeviceNFT contract uses latest openzeppelin version.

Reply:
Fixed